MerlinAU.sh

```diff
@@ -4885,8 +4910,10 @@ fi
 # menu setup variables #
 theExitStr="${GRNct}e${NOct}=Exit to Main Menu"
 theADExitStr="${GRNct}e${NOct}=Exit to Advanced Menu"
+theLGExitStr="${GRNct}e${NOct}=Exit to Logs Menu"
+
 padStr="      "
-SEPtop="----------------------------------------------------------"
+SEPstr="----------------------------------------------------------"
```

Comment:
FYI, I changed the variable name back since "`SEPtop`" was meant to indicate that it was for the **top** Main Menu only; but since it's now used everywhere, the old name "`SEPstr`" fits better, and is less confusing, IMO.

Reply:
Yeah that's totally fine I more or less assumed that's why you did it :)

Reply:
I actually thought about this last night why I didn't do it this way but I had already done it and it worked so I figured I'd leave it haha, at leave it gave you a demo of what I had pictured for the other menus